tests/rkt_image_rm_test.go

```diff
@@ -152,6 +242,24 @@ func TestImagePrepareRmRun(t *testing.T) {
 	spawnAndWaitOrFail(t, cmd, true)
 }
 
+func getImageName(ctx *testutils.RktRunCtx, name string) (string, error) {
```

Comment:
You could make this function a bit simpler - instead of returning an error, just take a `t *testing.T` and call `t.Fatalf("...")` if something gets wrong.